pdr_backend/data_eng/test/resources.py

```diff
@@ -60,6 +74,20 @@ def _data_ss(parquet_dir, input_feeds, st_timestr=None, fin_timestr=None):
     )
 
 
+# pylint: disable=line-too-long
+@enforce_types
+def _web3_pp():
```

Comment:
Why is this function being added in two places? DRY violation

Reply:
Because it was already in multiple places.

Now it's been restructured a bit as part of your re-write, hopefully making it easier to sort it out. I'll cleanup tests and references such that everything is using `mock_test_pp` from `web3_pp.py`

I believe `test_noganache/conftest.py` is still using it. I'll cleanup as appropriate.

Reply:
I have simplified most of the code that should use mock_ppss, and identified some new ones. There may be others.. I'm adding them here as I go so I can clean them up.
```
/util/test_noganache/test_checknetwork.py::test_check_network_main
```

Reply:
I updated this implementation, and further to use the common mock_ppss.

mock_ppss was using a non-url for both: subgraph_url and rpc_url, I have updated both so they are valid and cleaned up test_check_network_main()

Reply:
I believe I resolved this as much as possible, while trying to keep the PR focused.
Resolving this conversation.